src/cbor/decoder.rs

```diff
@@ -69,7 +69,8 @@ impl DecoderCursor {
     /// Read a byte string and return it.
     fn read_byte_string(&mut self) -> Result<CborType, CborError> {
         let length = self.read_int()?;
-        if length > usize::max_value() as u64 {
+        let limit = usize::max_value() as u64;
```

Comment:
I feel like we could use the limit introduced in #21 here.

Reply:
I think that's reasonable.